ansible/roles/boot-iso/tasks/supermicro.yml

```
@@ -55,6 +55,7 @@
     return_content: yes
 
 - name: SuperMicro - Set Boot
+  ignore_errors: yes
```

Might want to add a comment above explaining why we ignore errors here.  (Old firmware, but a workaround of pre-configuring your boot order for CD first and watching consoles to determine when to unmount the boot iso.)